.eslintrc

```diff
@@ -0,0 +1,20 @@
+{
+  "parser": "@typescript-eslint/parser",
+  "extends": [
+    "eslint:recommended",
+    "plugin:jest/recommended",
+    "plugin:jest-dom/recommended",
+    "plugin:prettier/recommended",
+    "plugin:@typescript-eslint/eslint-recommended",
+    "plugin:@typescript-eslint/recommended"
+  ],
+  "env": {
+    "es6": true,
+    "browser": true,
+    "node": true
+  },
+  "rules": {
+    "@typescript-eslint/no-empty-interface": "off",
```

Comment:
This allows us to use empty `interface` definitions that are only extending other interfaces. Example:

https://github.com/Developer-DAO/web3-ui/blob/f0c96df7468fbe0993b8e90979407c27ba7a22bd/packages/components/src/components/Provider/Provider.tsx#L4